### PR TITLE
1.2.x

### LINKED ISSUE #168: installation failure
Branch `1.2.x` fails to install with error: `pyrproj.exceptions.CRSError: Invalid projection: {"epsg":"4326"}....`

### PR DESCRIPTION
**Pull request by miccferr**
Fix #168

### CODE REVIEW

File: skmob/tessellation/tilers.py [hunk 1]
```diff
@@ -274,7 +267,6 @@ def _build(self, base_shape, meters, crs=constants.DEFAULT_CRS):
 
         # We work with the universal crs epsg:3857
         tmp_crs = constants.DEFAULT_CRS
```
Comment (gbarlacchi) [inline skmob/tessellation/tilers.py:269]: The value assigned to this variable is never. Can you please remove it?

File: skmob/utils/constants.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
 """
 Useful constants
 """
+from pyproj import CRS
 
-UNIVERSAL_CRS = {"epsg": "3857"}
-DEFAULT_CRS = {"epsg": "4326"}
+UNIVERSAL_CRS = CRS.from_epsg(3857) #{"epsg": "3857"}
```
Comment (gbarlacchi) [inline skmob/utils/constants.py:6]: Remove comments please.

Comment (miccferr) [reply skmob/utils/constants.py]: fix in f9ad264

File: skmob/tessellation/tilers.py (hunk 1 above)
Comment (miccferr) [reply skmob/tessellation/tilers.py]: fix in 558b121

**Pull request merged**